test/property.test.js

```diff
@@ -125,6 +292,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('fails to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
+      required: 'true',
+      defaultValue: 'X'
+    });
+
+    expect(propertyGenerator.run).to.throw(Error);
```

Comment:
When expecting a function to throw, then you should check that the expected error was thrown, so that the test fails when a different error is raised. Otherwise the code in test can trigger different error (like `TypeError: undefined is not a function`) and you will never notice.

I think it may be the case here, I would expect `propertyGenerator.run` to throw because you don't pass `propertyGenerator` as the `this` value here. If it turns out I am right, then I think you should fix this line to `expect(function() { propertyGenerator.run(); }).to.throw(/.../)`.